cmd/storage-rest-client.go

```diff
@@ -278,6 +292,8 @@ func (client *storageRESTClient) ReadFileStream(volume, path string, offset, len
 	values.Set(storageRESTFilePath, path)
 	values.Set(storageRESTOffset, strconv.Itoa(int(offset)))
 	values.Set(storageRESTLength, strconv.Itoa(int(length)))
+	globalNetStats.Receive(uint64(length))
+	defer globalNetStats.FinishReceiving(uint64(length))
```

Comment:
if err != nil here, then the rest call failed. No bytes were streamed. i.e. no inaccuracy in cancelling.

Reply:
Not sure I understand - won't this count this transfer even if it doesnt happen (i.e. when err != nil) or stops partway?